Resources/Prototypes/_LostParadise/Maps/Pools/NY_LostParadiseMapPool.yml

```
@@ -0,0 +1,11 @@
+- type: gameMapPool
+  id: NY_LostParadiseMapPool
+  maps:
+  - NY_LPPAsterisk
```

yamllint (1.35.1)

[error] wrong indentation: expected at least 3

(indentation)